samples/javascript_typescript/60.multilingual-luis-bot/typings/globals/mocha/index.d.ts

```diff
@@ -0,0 +1,230 @@
+// Generated by typings
```

Comment:
You can install this https://www.npmjs.com/package/@types/mocha as a dev dependency and remove this file